packages/lesswrong/components/messaging/ConversationPage.tsx

```diff
@@ -51,24 +51,26 @@ const ConversationPage = ({ conversationId, currentUser, classes }: {
   const showModInboxLink = userCanDo(currentUser, 'conversations.view.all') && conversation.moderator
 
   return (
-    <SingleColumnSection>
-      <div className={classes.conversationSection}>
-        <div className={classes.row}>
-          <Typography variant="body2" className={classes.backButton}><Link to="/inbox"> Go back to Inbox </Link></Typography>
-          {showModInboxLink && <Typography variant="body2" className={classes.backButton}>
-            <Link to="/moderatorInbox"> Moderator Inbox </Link>
-          </Typography>}
+    <div>
```

Comment:
Undo all changes in this file, including this extra div (just use Git or your IDE to revert the file)